osl/namespace_linux.go

```diff
@@ -607,7 +607,7 @@ func reexecSetIPv6() {
 	}
 
 	if err = ioutil.WriteFile(path, []byte{value, '\n'}, 0644); err != nil {
-		logrus.Errorf("failed to %s IPv6 forwarding for container's interface %s: %v", action, os.Args[2], err)
+		logrus.Warnf("failed to %s IPv6 forwarding for container's interface %s: %v", action, os.Args[2], err)
```

Comment:
Should we continue to `os.Exit` if the failure is not severe?

Reply:
Did you mean, remove the `os.Exit(4)` following this line, and let it continue on to the `os.Exit(0)` a few lines later?

Reply:
Yes. I am assuming no one is depending on that error exit code based on your observation that it is non-fatal?

Reply:
could we somehow skip it, instead of ignoring errors? https://github.com/docker/libnetwork/pull/2122#comment-375056700

Reply:
@thaJeztah from a quick parsing of the code looks like this function is potentially also used to disable ipv6, so most likely is always been called (but this has to be confirmed). If so what can be done is that if the path is not there and we were anyway disabling ipv6 then maybe we can avoid printing the log